.github/ISSUE_TEMPLATE/feature_work_template.md

```diff
@@ -20,6 +20,18 @@ _As a [type of user], I want to [do something] so that [this outcome]_
 ### Tasks
 - [ ] _Key pieces of work that will need to be done to meet the User Story and CoAs_
 
+### Definition of Done Checklist
+- [ ] Usability validated
+- [ ] Design QA passed
```

Comment:
Based on the extra definition of `Design QA passed`, I think it makes more sense to rename this DoD item to the following.

```suggestion
- [ ] UI works as designed
```